app/views/tag/_tagging.html.erb

```diff
@@ -48,7 +48,7 @@ $(function () {
 <% parent ||= nil %>
 
 <% if current_user && (parent != :profile || (current_user.id == user.id || logged_in_as(['admin']))) && !@preview  %>
-  <% if current_user.first_time_poster && !(current_user.username == @node.author.username || ( @node.coauthors && @node.coauthors.exists?(username: current_user.username)) || logged_in_as(['admin', 'moderator'])) %>
+  <% if current_user.first_time_poster && !(parent == :profile || current_user.username == @node.author.username || ( @node.coauthors && @node.coauthors.exists?(username: current_user.username)) || logged_in_as(['admin', 'moderator'])) %>
```

Comment:
hi @17sushmita ,  maybe we could try use nil navigator `@node&.author&.username`  https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

Reply:
Hi @cesswairimu, Actually the `@node` even if present would be referring to the nodes written by the user. I think this tag moderation is relevant to only notes page and not the profile page.
If we also do tag moderation to the profile page, new user would not be able to add tags to his own profile. e.g-
![image](https://user-images.githubusercontent.com/42088159/124771232-97184b80-df58-11eb-98d2-e4d2fad07f80.png)

And for nodes page, `@node` will never be nil, so the nil navigator will not be required.
Did I miss any of the case? What do you think?

Reply:
Aha! gotcha, thanks for the clarification @17sushmita, I will try to start rebuild

Reply:
Looks like we are all good here.. let me know if we are good to merge. Thanks

Reply:
yes @cesswairimu. It is ready to merge.